src/rocksdb.rs

```diff
@@ -38,29 +38,12 @@ use std::rc::Rc;
 use std::str::from_utf8;
 use std::sync::Arc;
 use std::{fs, ptr, slice};
+use write_batch::{CFHandle, WriteBatch};
 
 use table_properties::{TableProperties, TablePropertiesCollection};
 use table_properties_rc::TablePropertiesCollection as RcTablePropertiesCollection;
 use titan::TitanDBOptions;
 
-pub struct CFHandle {
-    inner: *mut DBCFHandle,
-}
-
-impl CFHandle {
-    pub fn id(&self) -> u32 {
-        unsafe { crocksdb_ffi::crocksdb_column_family_handle_id(self.inner) }
-    }
-}
-
-impl Drop for CFHandle {
-    fn drop(&mut self) {
-        unsafe {
-            crocksdb_ffi::crocksdb_column_family_handle_destroy(self.inner);
-        }
-    }
-}
-
```

Comment:
`CFHandle` isn't specific to `WriteBatch` so I don't think it should be moved into `write_batch.rs` Can you move these definitions back here?

Reply:
Ok. I just do not want to import rocksdb mod in write_batch.rs